### PR TITLE
Destination S3 - connector check ignores the s3_bucket_path

### PR DESCRIPTION
**Pull request by haithem-souala**
Fix for #15195 & #11871

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@sashaNeshcheret can you review this contribution?

**Comment by haithem-souala** [comment]
Hey @sashaNeshcheret, could you review this PR please?

**Comment by marcosmarxm** [comment]
@sashaNeshcheret can you do the final approval here?

**Comment by sashaNeshcheret** [comment]
Hi @haithem-souala, sorry i've not a chance to make review earlier. 
I've tested PR and it really covers case when user has just read permissions on s3 and in such case we are trying to create new object(with bucket name) instead of using path that specified in config. From my perspective it can be properly published and merged. Thanks for your work.
CC: @marcosmarxm

**Review by sashaNeshcheret — APPROVED** [review]
Looks good to me

**Comment by haithem-souala** [comment]
Hey @marcosmarxm,
The PR is approved, how can i publish the connector ?

File: airbyte-integrations/connectors/destination-s3/Dockerfile [hunk 1]
```diff
@@ -41,5 +41,5 @@ RUN /bin/bash -c 'set -e && \
        echo "unknown arch" ;\
     fi'
 
-LABEL io.airbyte.version=0.3.13
-LABEL io.airbyte.name=airbyte/destination-s3
+LABEL io.airbyte.version=0.3.14
+LABEL io.airbyte.name=airbyte/destination-s3
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/destination-s3/Dockerfile:45]: add eof here and allow maintaners edit your branch so I can run the publish command.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**